apps/stable_diffusion/src/models/opt_params.py

```diff
@@ -60,7 +71,6 @@ def get_params(bucket_key, model_key, model, is_tuned, precision):
 
 
 def get_unet():
-    variant, version = hf_model_variant_map[args.hf_model_id]
```

Comment:
This is necessary to not fix the version and variant when executing it multiple times.